docs/book/v1/manual.md

```diff
@@ -29,7 +29,10 @@ The following details the constructor of the `Zend\Expressive\Session\Cache\Cach
 public function __construct(
     \Psr\Cache\CacheItemPoolInterface $cache,
     string $cookieName,
+    string $cookieDomain = null,
     string $cookiePath = '/',
+    bool $cookieSecure = false,
+    bool $cookieHttpOnly = false,
```

Comment:
While I understand the order you're using here, it breaks backwards compatibility to inject them here instead of at the end. Let's put all the new options at the end, and we can make a `@todo` notation in the constructor docblock to change the order in the next major version.